core-api-library/library/src/main/java/net/gini/android/core/api/requests/Exceptions.kt

```diff
@@ -0,0 +1,6 @@
+package net.gini.android.core.api.requests
+
+import java.io.IOException
+
+class ApiException(message: String) : IOException(message)
+class NoInternetException(message: String) : IOException(message)
```

Comment:
Currently this exception is not thrown anywhere. Let's keep an eye on it and remove it later, if we don't use it.